packages/mui-codemod/src/deprecations/dialog-props/test-cases/expected.js

```diff
@@ -0,0 +1,78 @@
+import Dialog from '@mui/material/Dialog';
+import { Dialog as MyDialog } from '@mui/material';
+
+<Dialog
+  slots={{
+    transition: CustomTransition,
+    paper: CustomPaper
+  }}
+  slotProps={{
+    paper: { className: 'paper' },
+
```

Comment:
couldn't figure out why new line keep getting added, any help would be appreciated here

Reply:
I'm not sure, either. Would you know @siriwatknp?

Anyway, this should be covered by the user's formatting rules, as those might differ, and it would be impossible for us to accommodate all linting rules. So I wouldn't care too much until someone complains.